qoffer-domain/src/main/groovy/life/qbic/portal/portlet/products/ListProductsDataSource.groovy

```diff
@@ -10,7 +11,7 @@ import life.qbic.portal.portlet.exceptions.DatabaseQueryException
  *
  * @since 1.0.0
  */
-interface ListPackagesDataSource {
+interface ListProductsDataSource {
 
   /**
    * Queries a data source for all available service
```

Comment:
Documentation is still talking about packages instead of products

Reply:
Thanks for trying but it's not part of the changes so i can't suggest anything 😿